src/aqueduct-helm/templates/persistentvolume.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: aqueduct-pv
+spec:
+  storageClassName: aqueduct
+  capacity:
+    storage: 5Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: "/root/.aqueduct"
```

Comment:
It seems from here that `hostPath` is only used for dev and testing for single-node cluster? Does this work for multi-node clusters?

Reply:
Yep I tested this on multi-node clusters as well.